packages/sanity/src/structure/panes/document/document-layout/DocumentLayout.tsx

```diff
@@ -85,6 +88,9 @@ export function DocumentLayout() {
   const zOffsets = useZIndex()
   const previewUrl = usePreviewUrl(value)
 
+  const createLinkMetadata = getCreateLinkMetadata(value)
+  const CreateLinkedBanner = useSanityCreateConfig().components?.documentLinkedBanner
```

Comment:
This is done to avoid having to put component code belonging to the Create integration in the structure package.